src/Symfony/Component/Console/Command/LazyCommand.php

```diff
@@ -22,18 +22,12 @@
  */
 final class LazyCommand extends Command
 {
-    private $command;
```

Comment:
this change is even broken, because you turned `$command` into a typed property with the `Closure` type, which is not the current expectation of the code (the type of this property is `\Closure|Command`, see `getCommand()`)

Reply:
Thanks, will fix that in Rector :+1: